### PR TITLE
Update config defaults

### PR DESCRIPTION
**Pull request by gammazero**
- Fixes issue #131
- Fixes issue #134

### CODE REVIEW

**Review by willscott — APPROVED** [review]

File: config/ingest.go [hunk 1]
```diff
@@ -2,18 +2,20 @@ package config
 
 const (
 	defaultLinkCacheSize   = 1024
-	defaultLinkedChunkSize = 100
+	defaultLinkedChunkSize = 16384
```
Comment (masih) [inline config/ingest.go:5]: For the readers of code, can I ask you to add a comment explaining why this number?

Comment (gammazero) [reply config/ingest.go:5]: Done

**Comment by masih** [comment]
@gammazero  Can I ask you to set the version to 0.3.0 please if all of it is happening in this PR.

**Comment by masih** [comment]
> @gammazero Can I ask you to set the version to 0.3.0 please if all of it is happening in this PR.

Discussed via DM, since new function is being exported, no API changes hence the 0.2.1

**Pull request merged**